blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/MatrixProjectImpl.java

```diff
@@ -29,32 +31,32 @@
 
     private final MatrixProject matrixProject;
 
-    public MatrixProjectImpl(MatrixProject folder, Link parent) {
-        super(folder, parent);
+    public MatrixProjectImpl(BlueOrganization organization, MatrixProject folder, Link parent) {
+        super(organization, folder, parent);
         this.matrixProject = folder;
     }
 
     @Extension(ordinal = 1)
     public static class PipelineFactoryImpl extends BluePipelineFactory{
 
         @Override
-        public MatrixProjectImpl getPipeline(Item item, Reachable parent) {
+        public MatrixProjectImpl getPipeline(Item item, Reachable parent, BlueOrganization organization) {
```

Comment:
Something I've found is that in many cases the `Reachable parent` it is already the organization and that we may just change the type or check if it is an organization. Un less we want to support hierarchies in the future.
Have you thought about it? Did I get it wrong?

See: https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractMultiBranchCreateRequest.java#L61